packages/apollo/README.md

```diff
@@ -642,6 +643,10 @@ When using the codegen command with Typescript or Flow, make sure to add the `__
 
 If you're using a client like `apollo-client` that does this automatically for your GraphQL operations, pass in the `--addTypename` option to `apollo codegen:generate` to make sure the generated Typescript and Flow types have the `__typename` field as well. This is required to ensure proper type generation support for `GraphQLUnionType` and `GraphQLInterfaceType` fields.
 
+## Default `--includes` parameter
```

Comment:
I think this would be better under the `--includes` flag description. Same for excludes. You can find these flag descriptions in `packages/apollo/src/Command.ts`